tokio/src/io/async_read_futures.rs

```diff
@@ -1,4 +1,4 @@
-use crate::io::AsyncRead;
+use super::AsyncRead;
```

Comment:
Does this file contain other `Read` futures in it?

Reply:
@LucioFranco Just the one, I named it in plural to allow for more. Want me to rename it to async_read_future.rs or something?

Reply:
It's internal, not a big deal. We can update in a follow up PR.

My preference would be to create `tokio::io::futures` as a dir and have all the futures in there.